src/states_screens/race_gui.cpp

```diff
@@ -1164,7 +1164,8 @@ void RaceGUI::drawLap(const AbstractKart* kart,
 
     static video::SColor color = video::SColor(255, 255, 255, 255);
     int hit_capture_limit =
-        race_manager->getHitCaptureLimit() != std::numeric_limits<int>::max()
+        (race_manager->getHitCaptureLimit() != std::numeric_limits<int>::max()
+         && race_manager->getHitCaptureLimit() != 0)
```

Comment:
I wonder in which case hit_capture_limit can be 0 ?

Reply:
The comment before, here used to not show tje trophy symbol that is used to display the hit capture limit if the limit is 0